### ISSUE TITLE
Cannot locate symbol "__gxx_personality_v0"

### ISSUE THREAD

**Issue by TPEcool** [geode-sdk/geode#1062]
### Geode Issue

- [X] I confirm that this crash is NOT related to a mod but directly to Geode Loader itself.

### Platform

Android32

### Geode Version

v1.4.1

### Mods Installed

Don't remember but half the mods were disabled due to game version

### Steps to Reproduce

1. Updated GD to 2.2.142
2. Updated Geode to 1.4.1
3. No longer works

### Crash Report

Crash report: https://hastebin.com/share/atebaheqog.php
Log: https://hastebin.com/share/exozezulav.makefile

**Comment by qimiko** [comment]
interesting file extensions

anyways, this is just having an outdated geode. go into the launcher settings, tap "open file browser" and delete the geode.android32.so and geode.android64.so files (depending on which exist). check for updates again, and it should redownload the correct versions

**Comment by TPEcool** [comment]
> delete the geode.android32.so and geode.android64.so files (depending on which exist). check for updates again, and it should redownload the correct versions

Tried it, didn't seem to have done anything. I then installed another version of GD and now it launches fine but some mods complain about being made for 2.205 but me using 2.206. The mod settings page also does not display some mods. For example:
- I can't find xdBot in the Download or Featured tabs
- A mod I installed, "RobTop Jumpscare", does not show up on the Installed tab
